packages/lego-editor/src/components/ImageMenu.tsx

```diff
@@ -0,0 +1,32 @@
+import { Editor } from '@tiptap/react';
+import styles from './ImageMenu.module.css';
+import { GalleryThumbnails, Trash } from 'lucide-react';
+import { ToolbarButton } from './Toolbar';
+
+type Props = {
+  editor: Editor | null;
+};
+
+export const ImageMenu = ({ editor }: Props) => {
+  const isImage = editor?.isActive('image');
+  const isFigure = editor?.isActive('figure');
+  if (!editor || !(isImage || isFigure)) return null;
+
+  return (
+    <div className={styles.root}>
+      <ToolbarButton
```

Comment:
Does this have a tooltip?

Reply:
I'll add a tooltip in a separate PR. It requires some refactoring as the `Tooltip`-component is currently in `lego-webapp`